From 3f9ef50f47292d6cfa383d26f38643d8a3a566ae Mon Sep 17 00:00:00 2001
From: Kill3rEz <98993017+Kill3rEz@users.noreply.github.com>
Date: Wed, 17 Jun 2026 15:58:19 +0200
Subject: [PATCH 2/9] Add Oplus user data member to StagefrightRecorder

Add mOplusUserData field for OPLUSHDR support.
OnePlus/OPPO patch adding a private member to store HDR-related video recording data.

(cherry picked from commit 5a6275c3760caf42313a1bfef2719b21257561fd)
---
 media/libmediaplayerservice/StagefrightRecorder.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/media/libmediaplayerservice/StagefrightRecorder.h b/media/libmediaplayerservice/StagefrightRecorder.h
index ee5a1d530b..126c3f74b1 100644
--- a/media/libmediaplayerservice/StagefrightRecorder.h
+++ b/media/libmediaplayerservice/StagefrightRecorder.h
@@ -114,6 +114,7 @@ private:
     void updateMetrics();
 
     AString mLogSessionId;
+    AString mOplusUserData;  // [PATCH OPLUSHDR]
     audio_source_t mAudioSource;
     privacy_sensitive_t mPrivacySensitive;
     video_source mVideoSource;
